### DISCUSSION TITLE
qrcode not being rendered in terminal

### DISCUSSION THREAD

**Discussion by 1neeol** [Auties00/WhatsappWeb4j#533] [Q&A]
I'm trying to do a simple implementation to see how the library works, but when trying to generate the qrcode in the terminal, the application is not rendering and finishing execution.

![code](https://github.com/user-attachments/assets/cd1c8dd9-7127-4b0b-a291-d2f704c3b5cc)

**Comment by Auties00** [comment]
You are missing awaitDisconnection()

**Reply by 1neeol** [reply]
A question off topic for this subject. Has support for multi connections already been implemented?

**Reply by Auties00** [reply]
Yeah sessions are independent

**Reply by 1neeol** [reply]
Can you give me an example of how to create new instances and how to manage these connection instances?
